go.mod

```diff
@@ -2,6 +2,9 @@ module github.com/qdm12/ddns-updater
 
 go 1.20
 
+//required for local testing. Linter will not build the docker container with it in place though. 
+//replace github.com/qdm12/ddns-updater/internal/provider/providers/nowdns => ./internal/provider/providers/nowdns
+
```

Comment:
I don't think this is needed even for local testing, since the Go module is `github.com/qdm12/ddns-updater` (even your local version), so it will use your local files (also the `internal` directory cannot be imported from outside)

```suggestion
```

Reply:
I get past the linter now, but the image fails to build. Hits me with this:

`Step 37/50 : RUN GOARCH="$(xcputranslate translate -targetplatform ${TARGETPLATFORM} -field arch)"     GOARM="$(xcputranslate translate -targetplatform ${TARGETPLATFORM} -field arm)"     go build -trimpath -ldflags="-s -w     -X 'main.version=$VERSION'     -X 'main.created=$CREATED'     -X 'main.commit=$COMMIT'     " -o app cmd/updater/main.go
 ---> Running in 66d5b79a6e51
go: invalid GOARM: must be 5, 6, 7
The command '/bin/sh -c GOARCH="$(xcputranslate translate -targetplatform ${TARGETPLATFORM} -field arch)"     GOARM="$(xcputranslate translate -targetplatform ${TARGETPLATFORM} -field arm)"     go build -trimpath -ldflags="-s -w     -X 'main.version=$VERSION'     -X 'main.created=$CREATED'     -X 'main.commit=$COMMIT'     " -o app cmd/updater/main.go' returned a non-zero code: 2
`

Reply:
Moving to main thread.